### DISCUSSION TITLE
Modifying existing Web Radio URL also removes the Cover art

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1252] [Fixed Issues]
Running rAudio 20230224

To reproduce problem:
     - Modify existing Web Radio URL
     - Cover art reverts back to default cover art
  
![image](https://user-images.githubusercontent.com/76575081/221428522-1d054624-8541-433a-ae87-8dde6cf2eb82.png)

Change cover art / add from file successfully applies the "lost" cover art.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by ceejayemm** [comment]
Jim

This may well be related to your other (#1253 ) posting.  Solve that and I think you solve this as well.

Chris

**Comment by Sagnasty** [comment]
Chris,

This problem occurred with shared data (other server) enabled and disabled.

Jim

**Comment by rern** [comment]
Fixed with today's update.
